.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Tell us about a problem you are experiencing
+---
+
+**What steps did you take and what happened:**
+[A clear and concise description on how to REPRODUCE the bug.]
+
+
+**What did you expect to happen:**
+
+
+**Anything else you would like to add:**
+[Miscellaneous information that will assist in solving the issue.]
+
+
+**Environment:**
+
+- Cluster-api version:
+- Cluster-api-provider-metal3 version:
+- Environment (metal3-dev-env or other):
```

Comment:
Template is for dev-env, so all bugs reported here must happen in dev-env. This question is not needed.